src/lib.rs

```diff
@@ -394,6 +394,30 @@ where
     runner.run(f);
 }
 
+/// Sets the `tag` field of the current task.
+pub fn set_tag(tag: u64) {
+    use runtime::execution::ExecutionState;
+    ExecutionState::set_tag(tag);
+}
+
+/// Gets the `tag` field of the current task.
+pub fn get_tag() -> u64 {
+    use runtime::execution::ExecutionState;
+    ExecutionState::get_tag()
+}
+
+/// Gets the current step count.
```

Comment:
Why are we exposing this?

Reply:
Not exposed anymore -- I was not aware of `context_switches()`